Formula/t/tartufo.rb

```diff
@@ -0,0 +1,74 @@
+class Tartufo < Formula
+  include Language::Python::Virtualenv
+
+  desc "Searches through git repositories for high entropy strings and secrets"
+  homepage "https://tartufo.readthedocs.io/en/stable/"
+  url "https://files.pythonhosted.org/packages/d5/ea/6248387282150270f1886d75111f776f43e694f488a3a1ea3b5b0d1195f1/tartufo-5.0.0.tar.gz"
+  sha256 "99ab6652cae6de295aeb31089e9ba27d66d0ad695af493d2d5cbc795397d1c84"
+  license "GPL-2.0-only"
+  head "https://github.com/godaddy/tartufo.git", branch: "main"
+
+  depends_on "libgit2"
+  depends_on "python@3.12"
+
```

Comment:
Should we be including a `uses_from_macos "libffi"` here as well, due to the `cffi` resource?

Reply:
It's not always necessary. In this case though, it looks like Linux is showing `libffi` as an indirect dependency with linkage, so this would be one of those cases where we should declare it. I'll push that now.